src/explore-education-statistics-frontend/src/modules/table-tool/components/TableToolShare.tsx

```diff
@@ -42,17 +45,34 @@ const TableToolShare = ({
     }
     setPermalinkLoading(true);
 
-    const { id } = await permalinkService.createPermalink(
-      {
+    let id;
```

Comment:
Would suggest adding a type param or default value so it isn't inferred as `any`